src/app/drive/components/DriveExplorer/DriveExplorer.tsx

```diff
@@ -300,6 +353,9 @@ const DriveExplorer = (props: DriveExplorerProps): JSX.Element => {
                       <FolderSimplePlus size={20} />
                       <p className="ml-3">{i18n.get('actions.upload.folder')}</p>
                     </MenuItem>,
+                    <MenuItem onClick={() => dispatch(uiActions.setIsNameCollisionDialogOpen(true))}>
+                      <p>Test name collision</p>
```

Comment:
Remove this testing menu item
<img width="281" alt="Screenshot 2023-01-18 at 16 57 33" src="https://user-images.githubusercontent.com/34165099/213224533-f1d04c25-fabb-4f12-84e5-c1e61d786446.png">